### ISSUE TITLE
Log errors that occur during ThriftSender's flush

### ISSUE THREAD

**Issue by pschichtel** [jaegertracing/jaeger-client-java#728]
This is a followup issue to #726 / #369.

Currently both the HttpSender and (once #726 is merged) the UdpSender can fail on every flush() operation while the jaeger agent is not available.

The suggested improvement is to log just the first exception and then log once it works again. That way all the necessary/useful information is logged, but the log is not flooded with redundant errors.

**Comment by pschichtel** [comment]
@jpkrohling From looking at the call stacks I assume that either `RemoteReporter.FlushCommand#execute()` or `RemoteReporter.QueueProcessor#run()` would be the right place to handle the error logging, right?

**Comment by jpkrohling** [comment]
I believe so. We are already doing some exception handling at `QueueProcessor#run` and recording a metric in case it fails, so, that would be my preference.

**Comment by pschichtel** [comment]
That's what I implemented in #729

**Comment by pschichtel** [comment]
Is there any plan for an upcoming release that will contain these changes?

**Comment by pavolloffay** [comment]
I can cut a patch release today

**Comment by pschichtel** [comment]
Thanks!

**Comment by pavolloffay** [comment]
The artifact should be available in the central
